src/main/java/org/galatea/starter/domain/IexHistoricalPricesDB.java

```diff
@@ -0,0 +1,131 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class IexHistoricalPricesDB {
+
+  @Id
+  @GeneratedValue (strategy = GenerationType.AUTO)
+  private long id;
+
+  private String symbol;
+  private BigDecimal close;
+  private BigDecimal high;
+  private BigDecimal low;
+  private BigDecimal open;
+  private long volume;
+  private String date;
+
+  protected IexHistoricalPricesDB() {}
```

Comment:
Why `protected`? `@NoArgsConstructor`?

Reply:
Since it is only there to interact with JPA (not with any of our systems) I set it as protected, which the "Accessing Data with JPA" recommended. I'll definitely change it to the @NoArgsConstructor though!